garak/generators/mistral.py

```diff
@@ -20,6 +21,45 @@ class MistralGenerator(Generator):
     ENV_VAR = "MISTRAL_API_KEY"
     DEFAULT_PARAMS = Generator.DEFAULT_PARAMS | {
         "name": "mistral-large-latest",
+        # Sampling parameters
+        "top_p": None,                    # Nucleus sampling (0.0-1.0)
+        "frequency_penalty": None,        # Penalize word repetition by frequency (0.0-2.0)
+        "presence_penalty": None,         # Penalize word repetition (0.0-2.0)
+        # Generation control
+        "n": None,                        # Number of completions to return
+        "stop": None,                     # Stop sequences (string or list)
+        "stream": False,                  # Whether to stream responses
+        # Response format
+        "response_format": None,          # Output format (text, json_object, json_schema)
+        # Tool/function calling
+        "tools": None,                    # List of available tools/functions
+        "tool_choice": None,              # Tool calling control (none, auto, any, required)
+        "parallel_tool_calls": None,      # Enable parallel function calling
+        # Safety and prompting
+        "safe_prompt": None,              # Inject safety prompt
+        "prompt_mode": None,              # Toggle reasoning mode
+        # Optimization
+        "prediction": None,               # Expected completion for optimization
```

Comment:
`garak` does not currently expect generator to support _tools_, _safety prompts_, etc. avoid offering `DEFAULT_PARAMS` for items the that do not have a consistent abstraction at this time.


```suggestion
```